### DISCUSSION TITLE
document blocks and their purpose

### DISCUSSION THREAD

**Discussion by AdilArslannn** [easylist/easylist#20055] [Q&A] [answered]
Hello,
I have been messing around with the whole document block rules but I cannot seem to notice any difference. They don't block the website when used in Adblock plus. So then what is the purpose of those rules? 

Example document block rules from `easylist_adservers.txt`

`! $document blocks`
`||123-stream.org^$document`
`||1winpost.com^$document`
`||20trackdomain.com^$document`
`||2ltm627ho.com^$document`

**Comment by Yuki2718** [comment, marked as answer]
Adblock plus doesn't support document block, while uBlock Origin and Adguard do.
